lr-cli/cortx_setup/commands/pillar_sync.py

```diff
@@ -51,8 +51,11 @@ def run(self, **kwargs):
         conf_path = str(PRVSNR_FACTORY_PROFILE_DIR / 'confstore')
         # backup local consftore data
         self.logger.debug(f"Copy local confstore file to {conf_path}")
-        state = 'components.provisioner.confstore'
-        StatesApplier.apply([state])
+        state_create = 'components.provisioner.confstore_dir_create'
```

Comment:
Lets change this to:
`conf_create = 'components.provisioner.confstore_create'`

Reply:
changed